model.py

```diff
@@ -5795,6 +5804,32 @@ class Resource(Base):
     representation_id = Column(
         Integer, ForeignKey('representations.id'), index=True)
 
+    # The rights status of this Resource.
+    rights_status_id = Column(Integer, ForeignKey('rightsstatus.id'))
+
+    # An optional explanation of the rights status.
+    rights_explanation = Column(Unicode)
+
+    # Many derivatives may be derived from a Resource.
+    derivative_derivations = relationship(
```

Comment:
This is a terrible name.

Reply:
Would it help if we said 'ResourceTransformation' instead of 'ResourceDerivation'? Then a Resource would be derived through a transformation rather than through a derivation, and this could be called `transformations`.